sklearn/cluster/_dbscan.py

```diff
@@ -179,27 +179,27 @@ class DBSCAN(ClusterMixin, BaseEstimator):
         .. versionadded:: 0.17
            metric *precomputed* to accept precomputed sparse matrix.
 
-    metric_params : dict, optional
+    metric_params : dict, default=None
         Additional keyword arguments for the metric function.
 
         .. versionadded:: 0.19
 
-    algorithm : {'auto', 'ball_tree', 'kd_tree', 'brute'}, optional
+    algorithm : {'auto', 'ball_tree', 'kd_tree', 'brute'}, default='auto'
         The algorithm to be used by the NearestNeighbors module
         to compute pointwise distances and find nearest neighbors.
         See NearestNeighbors module documentation for details.
 
-    leaf_size : int, optional (default = 30)
+    leaf_size : int, default=30
         Leaf size passed to BallTree or cKDTree. This can affect the speed
         of the construction and query, as well as the memory required
         to store the tree. The optimal value depends
         on the nature of the problem.
 
-    p : float, optional
+    p : float, default=None
```

Comment:
This is more a comment to the reviewers than the contributor: I find the change to default=None for a parameter like this extremely unhelpful. What does "None" mean in the context of "the power of the minkowski metric"? When the user chooses `metric='minkowski'`, how is this None interpreted?